config/_default/menus/menus.en.yaml

```diff
@@ -3379,6 +3379,11 @@ main:
     parent: security_platform
     identifier: security_notifications
     weight: 6
+  - name: Threat Intel
+    url: security/threat_intel
+    parent: security_platform
+    idenfier: security_threat_intel
+    weight: 12
```

Comment:
@datamarmot The weight means it'll go to the end of the list. But you've placed the markdown higher up.

Can you clarify if your intention is to have "Threat Intel" After Notifications, or at the end of the menu?